Normalize the five man pages that are in section 1m, adjusting them to
be in section 8. (It's noted that Ubuntu delivered these man pages in
section 1.)

This problem will be reported upstream as ncurses 6.0 (the latest released
version) still has these five man pages in section 1m.

--- ncurses-6.0-20170408/man/captoinfo.1m.orig	2017-04-14 08:07:18.251516286 +0000
+++ ncurses-6.0-20170408/man/captoinfo.1m	2017-04-14 08:08:46.044512816 +0000
@@ -27,8 +27,8 @@
 .\" authorization.                                                           *
 .\"***************************************************************************
 .\"
-.\" $Id: captoinfo.1m,v 1.26 2016/10/15 17:26:09 tom Exp $
-.TH @CAPTOINFO@ 1M ""
+.\" $Id: captoinfo.8,v 1.26 2016/10/15 17:26:09 tom Exp $
+.TH @CAPTOINFO@ 8 ""
 .ds n 5
 .ds d @TERMINFO@
 .SH NAME
@@ -180,14 +180,14 @@
 capabilities \fBmeml\fR (memory lock) and \fBmemu\fR (memory unlock).
 These will be discarded with a warning message.
 .SH NOTES
-This utility is actually a link to \fB@TIC@\fR(1M), running in \fI\-I\fR mode.
+This utility is actually a link to \fB@TIC@\fR(8), running in \fI\-I\fR mode.
 You can use other \fB@TIC@\fR options such as \fB\-f\fR and  \fB\-x\fR.
 .PP
 The trace option is not identical to SVr4's.
 Under SVr4, instead of following
 the \fB\-v\fR with a trace level n, you repeat it n times.
 .SH SEE ALSO
-\fB@INFOCMP@\fR(1M),
+\fB@INFOCMP@\fR(8),
 \fBcurses\fR(3X),
 \fBterminfo\fR(\*n)
 .PP
--- ncurses-6.0-20170408/man/infocmp.1m.orig	2017-04-14 08:07:18.258235572 +0000
+++ ncurses-6.0-20170408/man/infocmp.1m	2017-04-14 08:09:44.499115082 +0000
@@ -27,8 +27,8 @@
 .\" authorization.                                                           *
 .\"***************************************************************************
 .\"
-.\" $Id: infocmp.1m,v 1.65 2017/02/18 19:21:05 tom Exp $
-.TH @INFOCMP@ 1M ""
+.\" $Id: infocmp.8,v 1.65 2017/02/18 19:21:05 tom Exp $
+.TH @INFOCMP@ 8 ""
 .ie \n(.g .ds `` \(lq
 .el       .ds `` ``
 .ie \n(.g .ds '' \(rq
@@ -560,12 +560,12 @@
 To see only the
 4.4BSD set, use \fB\-r\fR \fB\-RBSD\fR.
 .SH BUGS
-The \fB\-F\fR option of \fB@INFOCMP@\fR(1M) should be a \fB@TOE@\fR(1M) mode.
+The \fB\-F\fR option of \fB@INFOCMP@\fR(8) should be a \fB@TOE@\fR(8) mode.
 .SH SEE ALSO
-\fB@CAPTOINFO@\fR(1M),
-\fB@INFOTOCAP@\fR(1M),
-\fB@TIC@\fR(1M),
-\fB@TOE@\fR(1M),
+\fB@CAPTOINFO@\fR(8),
+\fB@INFOTOCAP@\fR(8),
+\fB@TIC@\fR(8),
+\fB@TOE@\fR(8),
 \fBcurses\fR(3X),
 \fBterminfo\fR(\*n).
 .sp
--- ncurses-6.0-20170408/man/infotocap.1m.orig	2017-04-14 08:07:18.264038382 +0000
+++ ncurses-6.0-20170408/man/infotocap.1m	2017-04-14 08:10:23.248357967 +0000
@@ -27,8 +27,8 @@
 .\" authorization.                                                           *
 .\"***************************************************************************
 .\"
-.\" $Id: infotocap.1m,v 1.12 2016/10/15 17:26:09 tom Exp $
-.TH @INFOTOCAP@ 1M ""
+.\" $Id: infotocap.8,v 1.12 2016/10/15 17:26:09 tom Exp $
+.TH @INFOTOCAP@ 8 ""
 .ds n 5
 .ds d @TERMINFO@
 .SH NAME
@@ -65,8 +65,8 @@
 You can use other \fB@TIC@\fR options such as \fB\-f\fR and  \fB\-x\fR.
 .SH SEE ALSO
 \fBcurses\fR(3X),
-\fB@TIC@\fR(1M),
-\fB@INFOCMP@\fR(1M),
+\fB@TIC@\fR(8),
+\fB@INFOCMP@\fR(8),
 \fBterminfo\fR(\*n)
 .PP
 This describes \fBncurses\fR
--- ncurses-6.0-20170408/man/tic.1m.orig	2017-04-14 08:07:18.270671602 +0000
+++ ncurses-6.0-20170408/man/tic.1m	2017-04-14 08:10:56.913329400 +0000
@@ -26,8 +26,8 @@
 .\" authorization.                                                           *
 .\"***************************************************************************
 .\"
-.\" $Id: tic.1m,v 1.63 2017/01/14 19:50:09 tom Exp $
-.TH @TIC@ 1M ""
+.\" $Id: tic.8,v 1.63 2017/01/14 19:50:09 tom Exp $
+.TH @TIC@ 8 ""
 .ie \n(.g .ds `` \(lq
 .el       .ds `` ``
 .ie \n(.g .ds '' \(rq
@@ -147,7 +147,7 @@
 \fB\-C\fR
 Force source translation to termcap format.
 Note: this differs from the \fB\-C\fR
-option of \fB@INFOCMP@\fR(1M) in that it does not merely translate capability
+option of \fB@INFOCMP@\fR(8) in that it does not merely translate capability
 names, but also translates terminfo strings to termcap format.
 Capabilities
 that are not translatable are left in the entry under their terminfo names
@@ -453,10 +453,10 @@
 \fB\*d/?/*\fR
 Compiled terminal description database.
 .SH SEE ALSO
-\fB@INFOCMP@\fR(1M),
-\fB@CAPTOINFO@\fR(1M),
-\fB@INFOTOCAP@\fR(1M),
-\fB@TOE@\fR(1M),
+\fB@INFOCMP@\fR(8),
+\fB@CAPTOINFO@\fR(8),
+\fB@INFOTOCAP@\fR(8),
+\fB@TOE@\fR(8),
 \fBcurses\fR(3X),
 \fBterm\fR(\*n).
 \fBterminfo\fR(\*n).
--- ncurses-6.0-20170408/man/toe.1m.orig	2017-04-14 08:07:18.273570835 +0000
+++ ncurses-6.0-20170408/man/toe.1m	2017-04-14 08:11:39.233809132 +0000
@@ -26,8 +26,8 @@
 .\" authorization.                                                           *
 .\"***************************************************************************
 .\"
-.\" $Id: toe.1m,v 1.29 2017/02/18 17:10:26 tom Exp $
-.TH @TOE@ 1M ""
+.\" $Id: toe.8,v 1.29 2017/02/18 17:10:26 tom Exp $
+.TH @TOE@ 8 ""
 .ie \n(.g .ds `` \(lq
 .el       .ds `` ``
 .ie \n(.g .ds '' \(rq
@@ -97,7 +97,7 @@
 showing \fB@TOE@\fR's progress.
 .IP
 The optional parameter \fIn\fR is a number from 1 to 10,
-interpreted as for \fB@TIC@\fR(1M).
+interpreted as for \fB@TIC@\fR(8).
 If ncurses is built without tracing support, the optional parameter is ignored.
 .TP
 \fB\-V\fR
@@ -108,10 +108,10 @@
 \fB\*d/?/*\fR
 Compiled terminal description database.
 .SH SEE ALSO
-\fB@TIC@\fR(1M),
-\fB@INFOCMP@\fR(1M),
-\fB@CAPTOINFO@\fR(1M),
-\fB@INFOTOCAP@\fR(1M),
+\fB@TIC@\fR(8),
+\fB@INFOCMP@\fR(8),
+\fB@CAPTOINFO@\fR(8),
+\fB@INFOTOCAP@\fR(8),
 \fBcurses\fR(3X),
 \fBterminfo\fR(\*n).
 .PP
